packages/lit-dev-content/site/tutorials/content/reactivity/02.md

```diff
@@ -0,0 +1,41 @@
+How does a reactive property work? When you add a reactive property, Lit adds [accessors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_accessors) for that property. When you set the property, Lit checks to see if the property has changed. If the property has changed, Lit starts an update cycle.
+The update cycle is _asynchronous_, so that updating several properties in the course of a single event handler, for example, only triggers a single update.
+
+By default, Lit uses a strict inequality check (`!==`) to determine whether a property has changed. If you mutate an existing object or array, Lit won't detect a change. Try clicking the "x" button next to one of the list items in the preview pane. You'll notice the list doesn't update.
+
+Lit's default equality check works great when using immutable data patterns. For example:
+
+```js
+this.array = [...this.array, newThing]
+this.obj = {...this.obj, id: ++this.obj.id}
+```
+
+To use this approach in the current component, replace the existing `splice()` call with the following code:
+
+```ts
+    this.groceries = [...this.groceries.slice(0, i),
+                      ...this.groceries.slice(i+1)];
+```
+
+This creates a new array containing all of the members from the first array, except for the one you want to remove. Since `groceries` is now a new array, the change triggers an update.
+
+{% aside "info" %}
+
+Immutable data can be tricky.
+
+The immutable approach looks simple with an array of strings, but correctly cloning a deeply-nested object can be complicated. If you're already using an immutable data pattern, it should just work with Lit. If you're not familiar with immutable data, you may want to starting with a library that handles immutable data.
+
+{% endaside %}
+
+Sometimes you may need to mutate data and trigger an update as a result. That's what `requestUpdate()` is for.
```

Comment:
We mention 2 ways to do this but we don't suggest when you'd want to use one way v. the other. Is that an issue? It's sort of out of scope, but are there any docs we can link to about this? I think it's fine as is, but just wanted to mention it in case there's a quick fix.

Reply:
Removed mention of `requestUpdate()`, linked to new section.